pydocx/export/html.py

```diff
@@ -361,6 +361,114 @@ def export_paragraph_property_indentation(self, paragraph, results):
 
         return results
 
+    def get_previous_level_paragraph(self, num_id, level_id):
+        level_id = int(level_id)
+
+        while True:
+            if level_id == 0:
+                prev_level_id = level_id
+            else:
+                prev_level_id = level_id - 1
+
+            prev_level_paragraphs = self.numbering_level_listing_track[num_id][prev_level_id]
+            if prev_level_paragraphs:
+                return prev_level_paragraphs[-1]
+
+            if prev_level_id == 0 and not prev_level_paragraphs:
+                # this is an edge case with older version of word when it may contain a sublist
```

Comment:
Thanks for the "why" comment! Can we capitalize the `t` in `This` to give it the treatment of a proper english sentence?